allensdk/brain_observatory/behavior/data_objects/eye_tracking/eye_tracking_table.py

```diff
@@ -220,12 +220,18 @@ def from_data_file(cls, data_file: EyeTrackingFile,
             sync_line_label_keys=Dataset.EYE_TRACKING_KEYS,
             trim_after_spike=False)
 
+        # mvr experiment fix, see notes in process_eye_tracking_data()
```

Comment:
Let's factor this code out into a separate function

```
def determine_if_mvr()
```
That accepts whatever arguments it needs and returns `True` if the experiment is an MVR experiment and `False` otherwise. That way, if we ever decide to change the logic for determining if something is an eye tracking file, we just have to change that function.